HoloRepositoryDemoApplication/Assets/HoloRpository/Scripts/StorageConnectionServer.cs

```diff
@@ -0,0 +1,256 @@
+            }
+            catch (Exception e)
+            {
+                Debug.LogError(e.Message);
+            }
+
+            if (!response.Successful)
+            {
+                Debug.LogError($"Failed to get glb model from {GetHologramUri}");
+                return;
+            }
+
+            var gltfObject = GltfUtility.GetGltfObjectFromGlb(response.ResponseData);
+
+            try
+            {
+                GameObject loadedObject = await gltfObject.ConstructAsync();
+                ModelSetting.Initialize(loadedObject);
+            }
+            catch (Exception e)
+            {
+                Debug.LogError($"{e.Message}\n{e.StackTrace}");
+                return;
+            }
+        }
+        #endregion Public Method
+
+        #region Private Common Method
+        private static IEnumerator GetRequest(string uri)
+        {
+            using (UnityWebRequest webRequest = UnityWebRequest.Get(uri))
+            {
+                yield return webRequest.SendWebRequest();
+                WebRequestReturnData = null;
+                if (webRequest.isNetworkError)
+                {
+                    Debug.Log("Web request Error! [Error message]: " + webRequest.error);
+                }
+                else
+                {
+                    WebRequestReturnData = webRequest.downloadHandler.text;
+                }
+            }
+        }
+
+        private static void CopyProperties(object source, object destination)
+        {
+            PropertyInfo[] destinationProperties = destination.GetType().GetProperties();
+            foreach (PropertyInfo destinationPi in destinationProperties)
+            {
+                PropertyInfo sourcePi = source.GetType().GetProperty(destinationPi.Name);
+                destinationPi.SetValue(destination, sourcePi.GetValue(source, null), null);
+            }
+        }
+
+        private static Patient JsonToPatient(JSONNode Json)
+        {
+            Patient patient = new Patient();
+
+            if (Json["pid"].Value == "")
+            {
+                Debug.Log("No response from server with this patient ID!");
+                return patient;
+            }
+
+            try
+            {
+                patient.pid = Json["pid"].Value;
+
+                PersonName name = new PersonName();
+                name.title = Json["name"]["title"].Value;
+                name.full = Json["name"]["full"].Value;
+                name.first = Json["name"]["first"].Value;
+                name.last = Json["name"]["last"].Value;
+                patient.name = name;
+
+                patient.gender = Json["gender"].Value;
+                patient.email = Json["email"].Value;
+                patient.phone = Json["phone"].Value;
+                patient.birthDate = Json["birthDate"].Value;
+                patient.pictureUrl = Json["pictureUrl"].Value;
+
+                Address address = new Address();
```

Comment:
I think Wenjie won't send address data from the Accessor anymore. But i think you may fix this when you update your code to Accessor API 1.0

Reply:
Yep, Currentlly, this one is based one the old version, actually, I have created the new class defination on my own pc, bcz it based on the UI server right now, use new defination will break the code, so I just keeped it right now